src/client.rs

```diff
@@ -586,7 +585,7 @@ fn save_and_notify_undelivered_messages(
                 } else if let Some(node_id) = user.node_id {
                     let result = AsyncClient::builder()
                         .timeout(Duration::from_secs(1))
-                        .build(&srv3.handle);
+                        .build();
```

Comment:
The async client uses its own event loop then?

Reply:
w/ the tokio reform, going forward you don't really pass Handles around. Now there's kind of a global event loop. Taskish things generally get an executor to schedule themself from a globalish CurrentThread executor or DefaultExecutor, I'm not entirely sure which one in this case.

But basically it should be running on the current executor (which sets itself as one of those when it starts)

Reply:
That's a lot of ish-ish! ;)

Reply:
very ish-ish hand waivey, I don't have a good grip on all this yet